Bottom: 99d82875244ebaf242476556720fffcfe7e81a24
Top:    72274864ab2fa3553d2cdfb306dae420573ff3ee
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:11:45 -0400

Refresh of cad-cah-smbc

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyGoComicsCom.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyGoComicsCom.java
index d31eb00..dcf84cf 100644
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyGoComicsCom.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyGoComicsCom.java
@@ -157,7 +157,6 @@ public class DailyGoComicsCom extends DailyComic {
             }
         }
 //Log.d("GO_Comics", "final_string " + final_str);
-        Log.d("DGCC", "final_string " + final_str);
     	final_str = final_str.replaceAll(".*=\"","");
         final_str = final_str.replaceAll("\"/*","");
